### PR TITLE
[Merged by Bors] - Monitor the server's status

### LINKED ISSUE #140: Monitor the server's status
Monitor the server's status which includes power on status and running status

### PR DESCRIPTION
**Pull request by wangzhenyu32**
Closes #140 

Signed-off-by: wangzhenyu32 <wang.zhenyu32@zte.com.cn>

### CODE REVIEW

File: adlik_serving/clients/python/BUILD [hunk 1]
```diff
@@ -3,6 +3,7 @@ sh_binary(
     srcs = ["build_pip_package.sh"],
     data = [
         "setup.py",
+        "//adlik_serving/apis:server_status_service_proto_py_pb2",
```
Comment (shiyingjin) [inline adlik_serving/clients/python/BUILD:6]: Should format in order

File: adlik_serving/clients/python/sever_status_client.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    start = time.time()
+    response = stub.serverStatus(status_request)
+    print(response)
+    end = time.time()
+    print('Status response is: \n{}'.format(json_format.MessageToJson(response)))
+    print('Running Time: {}s'.format(end - start))
+
+
+def _main():
+    _grpc_query_server_status()
```
Comment (shiyingjin) [inline adlik_serving/clients/python/sever_status_client.py:30]: No need the "_main" funciton

File: adlik_serving/server/grpc/BUILD [hunk 3]
```diff
@@ -75,6 +85,7 @@ serving_cc_library(
         ":grpc_service",
         ":task_service",
         ":grpc_model_operate_service",
+        ":grpc_server_status_service",
```
Comment (shiyingjin) [inline adlik_serving/server/grpc/BUILD:88]: Should format

**Comment by EFanZh** [comment]
bors d=shiyingjin

**Pull request closed without merging**